test/ThemeProvider_test.jsx

```diff
@@ -9,7 +9,7 @@ describe('<ThemeProvider>', () => {
     const wrapper = shallow(
       <ThemeProvider name="tropical">
         <div>Under the sea</div>
-      </ThemeProvider>
+      </ThemeProvider>,
```

Comment:
`npm outdated` doesn't report anything, but I can bump some version requirements in package.json.